packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/mappers/ReactViewGroupMapperTest.kt

```diff
@@ -87,7 +82,7 @@ internal class ReactViewGroupMapperTest {
         // Given
         whenever(mockReactViewGroup.background).thenReturn(mockColorDrawable)
         whenever(
-            mockDrawableUtils.getReactBackgroundFromDrawable(
+            this.mockDrawableUtils.getReactBackgroundFromDrawable(
```

Comment:
what is the need for `this`?

Reply:
Oh I guess it happened during refactoring, I'll remove it.